audio/src/main/java/com/application/audio/FileHelper.kt

```diff
@@ -0,0 +1,33 @@
+package com.application.audio
+
+import android.content.Context
+import java.io.File
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+class FileHelper {
+    /**
+     * Reads a WAV file as a FloatArray.
+     */
+    fun readWavAsFloatArray(context: Context, assetFileName: String?): FloatArray? {
```

Comment:
this needs to become better. Assuming 44 bytes for the header won't work in all cases. We need an algorithm which searches for the 'data' chunk in a WAV file and extracts that only.

Reply:
can be as a quick follow up PR.